gnosis/safe-react
# Feature: allow skipping execution when creating a transaction

## What it solves
Resolves #2839

## How this PR fixes it
Adds a checkbox "Execute transaction" in all tx review modals.

## How to test it
Try to make all kinds of transactions:
1. Safe Apps -> Confirm Tx
2. Contract Interaction -> Review
3. Contract Interaction -> Custom Tx -> Review
4. Send Collectibles -> Review
5. Send Funds -> Review

* As a single owner
* As one of the owners of a 1/x safe
* As one of the owners of a 2/x safe

## Screenshot
<img width="556" alt="Screenshot 2021-10-21 at 15 16 44" src="https://user-images.githubusercontent.com/381895/138288218-052182fb-8bb1-46fe-b4b8-454fad5387ac.png">
